README.md

```diff
@@ -71,8 +69,6 @@ We have many lint rules to help having a unify code all over the project. To exe
 yarn lint <project>
 ```
 
-> Running lint locally is slow and fill up heap memory. This is related to Typescript compilation and NX lint builder being slow. As a result you might get a `JavaScript heap out of memory`. NX is working on fixing this for an upcoming update. In the meantime you can do `NODE_OPTIONS=“--max-old-space-size=4096” yarn lint <project>` to raise the memory limit.
```

Comment:
@eirikurn That should be fixed with latest NX and node@14, right? at least locally I didn't get the error anymore

Reply:
Not sure about that. I still have some lint issues locally. IDE timeout at least. Might depend on which project you're lining, but haven't tried removing this from my .env yet.